src/Vizkit3dWorld.cpp

```diff
@@ -155,6 +164,14 @@ vizkit3d::RobotVisualization* Vizkit3dWorld::robotVizFromSdfModel(sdf::ElementPt
     //vizkit3d plugin with model defined in the sdf model
     vizkit3d::RobotVisualization* robotViz = new vizkit3d::RobotVisualization();
 
+    for (int i=0; i < excludeFromSdf.size(); ++i){
```

Comment:
Ok! I think I fixed it.

I've noticed that every time the property "frame" is changed in the RobotVisualization, a callback function checks if this visualizationFrame is part of any segment of the model. (not sure here, but I think that it defines the frame we see when we change the ViewerFrame on the vizkit widget)
So what happens is I was setting a visualisationFrame that wasn't even a valid segment, since I didn't load the sdfmodel to the RobotViz, and that's why the popup was called.
What I did is tot check